stdlib/ballerina-http/src/main/ballerina/ballerina/net/http/annotation.bal

```diff
@@ -14,6 +14,9 @@ package ballerina.net.http;
 @Field {value:"sslEnabledProtocols: SSL/TLS protocols to be enabled"}
 @Field {value:"ciphers: List of ciphers to be used"}
 @Field {value:"sslProtocol: The SSL protocol version"}
+@Field {value:"certificateRevocationVerifier: true/false to enable/disable certificate revocation verifier"}
```

Comment:
Shall we align the description with others? Eg The status of certificate revocation verifier {default value : true (enable)}